en/middleware/micrortps.md

```diff
@@ -34,6 +33,25 @@ The *Client* application is also compiled and built into the firmware as part of
 > **Tip** The bridge code can also be [manually generated](micrortps_manual_code_generation.md). Most users will not need to do so, but the linked topic provides a more detailed overview of the build process and can be useful for troubleshooting.
 
 
+## Supported uORB messages
```

Comment:
We should certainly explain the current state - and we should add a note saying that the list is in flux. But where the information is located should be based on whether users of the docs will rarely or always need the information. It sounds like you think most users will want to customise the list at this time?